.github/workflows/build-terraform-cli.yml

```diff
@@ -42,16 +42,14 @@ jobs:
       - uses: actions/setup-go@fac708d6674e30b6ba41289acaab6d4b75aa0753 # v4.0.1
         with:
           go-version: ${{ inputs.go-version }}
-      - name: Determine artifact basename
-        run: echo "ARTIFACT_BASENAME=${{ inputs.package-name }}_${{ inputs.product-version }}_${{ inputs.goos }}_${{ inputs.goarch }}.zip" >> $GITHUB_ENV
```

Comment:
@sarahethompson One more question (tysm for your help!) -- based on what you said about `actions-go-build` taking care of uploading the zip, from what I can tell in that case, it should no longer be necessary for us to determine the artifact name here and so I removed this step as well, but please let me know if you think this will cause any issues!

Reply:
Yes as far as I can tell you are totally right here!